server/src/embedded-languages/bash-support.ts

```diff
@@ -26,9 +34,14 @@ export const generateBashEmbeddedLanguageDoc = (analyzedDocument: AnalyzedDocume
     }
   })
   insertTextIntoEmbeddedLanguageDoc(embeddedLanguageDoc, 0, 0, shebang)
```

Comment:
here you'll be able to use embedded_bash_header directly and never touch it again.